src/ocean/time/Time.d

```diff
@@ -20,9 +20,9 @@
 
 module ocean.time.Time;
 
-import ocean.meta.types.Qualifiers;
-
+import core.time;
 import core.stdc.time: time_t;
+import ocean.meta.types.Qualifiers;
```

Comment:
Minor: probably best not to mix up style changes like the moving of `import ocean.meta.types.Qualifiers` with the functional changes this patch intends.

A question of taste: maybe use a selective import to make clear what `core.time` symbols you are using (I assume just `Duration`)?  And is there any reason not to use a nested import inside `TimeSpan` ... ?